strr-web/interfaces/property-manager-i.ts

```diff
@@ -0,0 +1,26 @@
+export interface PropertyManagerContactI {
+    firstName: string | undefined
```

Comment:
Should be just `string` types for these and others.

Reply:
Fixed but this pattern is found in other code and would need to be addressed  in future.